src/Runner.Worker/Handlers/CompositeActionHandler.cs

```diff
@@ -26,65 +26,60 @@ public sealed class CompositeActionHandler : Handler, ICompositeActionHandler
 
         public async Task RunAsync(ActionRunStage stage)
         {
-            // Validate args.
```

Comment:
Use the settings gear above -\> \"Hide whitespace changes\" when reviewing this file